src/IsoformInspectorView/components/dendrogram.tsx

```diff
@@ -11,194 +11,43 @@ export const Dendrogram = ({
   width: any
   height: any
 }) => {
-  if (!model.nivoData) {
+  if (!model.nivoData || !model.clusterData?.clusters) {
     return null
   }
 
-  const offset = model.top
-  let iter = 0
-  let refArr: Array<number>
-  const cellHeight = (height - model.top) / model.nivoData.data.length
-  let secondBranch = 0
+  const clust = cluster()
+    .size([height - 150, width])
+    .separation(() => 1)
```

Comment:
just for reference, this code is based on here from msaview https://github.com/GMOD/react-msaview/blob/02618d48d6a48ecc686c1b1461d82186e45a244e/lib/src/model.ts#L642-L646